docs/maintainers/issue-management.md

```diff
@@ -7,6 +7,7 @@
 Issues are opened for many different reasons.
 We use the following labels for issue classifications:
 
+* `Issue-Announcement`: the issue is for discussing an [Announcement](https://github.com/PowerShell/Announcements)
```

Comment:
I agree with the label name. But I guess we need to change the color to dark yellow to not be confused with the red Issue-problems.
We could pin the Announce repo in https://github.com/PowerShell/

Also I suggest add two link in main Readme.md in https://github.com/PowerShell/PowerShell#new-to-powershell
- link to Announcement repo https://github.com/PowerShell/Announcements
- link to discussions https://github.com/PowerShell/PowerShell/issues?q=is%3Aissue+is%3Aopen+label%3AIssue-Announcement

Reply:
I'm ok with special casing this label to be a different color than the other `Issue` types because of it's special nature.  I'll have the committee review this.

@iSazonov feel free to email me